{"uuid":"9eacdf4a-560b-4474-a184-5a3525645dc8","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["922915ac-f5df-481f-a9e1-9dfab19a040f"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884544112,"stop":1696884544112},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884544186,"stop":1696884544186},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884544247,"stop":1696884544248},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884546286,"stop":1696884546289}],"start":1696884543939,"stop":1696884546291}